install_config/web_console_customization.adoc

```diff
@@ -688,6 +688,51 @@ endif::[]
 ----
 
 endif::[]
+[[configuring-quota-notification-messages]]
+== Configuring Quota Notification Messages
+
+Whenever a user reaches or exceedes a quota a quota notification is put into the notificaiton drawer.
```

Comment:
exceeds a quota, a quota notification

Reply:
Hi @ahardin-rh, the notification will actually be generated whenever a user reaches a quota or exceeds a quota.  The notification message will be different for each.  I will correct the spelling of "exceeds"  -thanks

Reply:
Maybe just "Whenever a user reaches a quota"

Reply:
Ok, updated.